.circleci/continue_config.yml

```diff
@@ -61,7 +61,7 @@ executors:
   build:
     docker:
       # These images are managed by the Presto Release team.
-      - image: public.ecr.aws/oss-presto/prestocpp:root-20240416
+      - image: docker.io/majetideepak4/prestocpp:root-20240527
```

Comment:
After https://github.com/prestodb/presto/pull/22806 is merged, you can use the new dependency image - `public.ecr.aws/oss-presto/presto-native-dependency:latest`

See: https://us-east-1.console.aws.amazon.com/ecr/repositories/public/932483864676/presto-native-dependency?region=us-east-1